Src/SDK/Common/Temporal.Common.Payloads/internal/PayloadContainersEnumerableJsonConverter.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
```

Comment:
I have removed System.Text.Json in the branch where I started the CI work for that reason. However, I prefer it to Newtonsoft because it is updated, and it is an order of magnitude faster for larger payloads. Unfortunately, it has issues with serializing derived classes and dictionaries, so it probably isn't the best option.

Reply:
I also prefer System.Text.Json when I serialize _my_ data. But here, we are serializing the user's data. Types that potentially come from some library and that the user may not be able to modify. Newtonsoft has better coverage for data contact attributes that may affect the way those types are intended to be serialized. That is why I picked it. If in the future System.Text.Json catches up in that space, we can switch.